### PR TITLE
NodeJS cheat sheet

### PR DESCRIPTION
**Pull request by stanly-johnson**
**What does your Instant Answer do?**
A Cheat Sheet listing the commands in NodeJS

**What is the data source for your Instant Answer? (Provide a link if possible)**
Gosquared,
sourceUrl: https://www.gosquared.com/resources/node-cheat-sheet/

**Why did you choose this data source?**
Reliable and easy-to-understand

**Are there any other alternative (better) data sources?**
No

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
Programmers

**Is this Instant Answer connected to a DuckDuckHack Instant Answer idea?**
Yes, https://duck.co/ideas/idea/6177/nodejs-cheat-sheet

**Are you having any problems? Do you need our help with anything?**
Not yet

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
node2
node1

-----
IA Page: https://duck.co/ia/view/nodejs_cheat_sheet

### CODE REVIEW

File: share/goodie/cheat_sheets/json/nodejs.json [hunk 1]
```diff
@@ -0,0 +1,169 @@
+{
+    "id": "nodejs_cheat_sheet",
+    "name": "NodeJS",
+    "description": "NodeJS Cheat Sheet",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/nodejs.json]: Let's remove this, it’s a little redundtant

File: share/goodie/cheat_sheets/json/nodejs.json [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+        "Getting Started",
+        "Globals",
+        "stdio",
+        "Process",
+        "Util",
+        "Event EventEmitter",
+        "Net",
+        "Events"
+        ],
+  "sections": {
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/nodejs.json]: Please fix you indentation here and below.

**Comment by stanly-johnson** [comment]
@moollaza

**Comment by moollaza** [comment]
@stanly-johnson congrats! It's live: https://duckduckgo.com/?q=nodejs+cheat+sheet&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Comment by stanly-johnson** [comment]
@moollaza Thank You :)

**Pull request merged**